docs/metadata.md

```diff
@@ -55,19 +55,22 @@ The JSON file consists of a map from `name` to an entry object. Each entry objec
 - `entries`: if present then the entry is a directory. The value is a map from `name` to entry objects.
 - `perm`: the permissions to create the file with.
 
-The `content` and `entries` fields are mutually exclusive, it is an error to include both, 
-one or the other _must_ be present. 
+The `content` and `entries` fields are mutually exclusive, it is an error to include both,
+one or the other _must_ be present.
 The file or directory's name in each case is the same as the key which referred to that entry.
- 
+
 This hierarchy can then be used by individual containers, who can bind
 mount the config sub-directory into their namespace where it is
 needed.
 
 # Metadata image creation
 
-Run `linuxkit run` backends accept a `-data=STRING` option which will
-cause the given string to be passed to the VM in a platform specific
-manner to be picked up by the `pkg/metadata` component.
+`linuxkit run` backends accept two options to pass metadata to the VM in a platform specific
```

Comment:
It should really be "Some `linuxkit run` backends" as suggested in my original comment. Not all backends support metadata

Reply:
Ah. I had no idea what the original typo was.

Can we leave this and do a separate PR after? CI is halfway through, would like to get it complete.